
This file contains 2 patches to dist 3 (PL 51) that I used to generate
Configure for perl.

The first patch just cleans up and extends the $osvers detection for
DEC OSF/1 on the Alpha.

The second patch ensures that the ./mips file exists.  libpth.U calls
it.

	Andy Dougherty		doughera@lafcol.lafayette.edu


Index: Oldconfig.U
Prereq:  3.0.1.7 
*** mcon/U/Oldconfig.U	Thu Feb 16 09:52:38 1995
--- /home2/doughera/lib/dist/U/Oldconfig.U	Thu Feb 16 16:26:25 1995
***************
*** 264,275 ****
  			osvers="$3"
  			;;
  		osf1)	case "$5" in
! 				alpha)  osname=dec_osf
! 					case "$3" in
! 						[vt]1\.*) osvers=1 ;;
! 						[vt]2\.*) osvers=2 ;;
! 						[vt]3\.*) osvers=3 ;;
! 					esac
  					;;
  			hp*)	osname=hp_osf1	;;
  			mips)	osname=mips_osf1 ;;
--- 264,274 ----
  			osvers="$3"
  			;;
  		osf1)	case "$5" in
! 				alpha)
! ?X: DEC OSF/1 myuname -a output looks like:  osf1 xxxx t3.2 123.4 alpha
! ?X: where the version number can be either vn.n or tn.n.
! 					osname=dec_osf
! 					osvers=`echo "$3" | sed 's/^[vt]//'`
  					;;
  			hp*)	osname=hp_osf1	;;
  			mips)	osname=mips_osf1 ;;
Index: usrinc.U
Prereq:  3.0.1.1 
*** mcon/U/usrinc.U	Sun May  8 22:14:36 1994
--- /home2/doughera/lib/dist/U/usrinc.U	Tue Feb 21 11:00:10 1995
***************
*** 60,71 ****
  	fi
  	$rm -f usr.c usr.out
  	echo "and you're compiling with the $mips_type compiler and libraries."
  else
  	echo "Doesn't look like a MIPS system."
  	echo "exit 1" >mips
- 	chmod +x mips
- 	$eunicefix mips
  fi
  echo " "
  case "$usrinc" in
  '') ;;
--- 60,72 ----
  	fi
  	$rm -f usr.c usr.out
  	echo "and you're compiling with the $mips_type compiler and libraries."
+ 	echo "exit 0" >mips
  else
  	echo "Doesn't look like a MIPS system."
  	echo "exit 1" >mips
  fi
+ chmod +x mips
+ $eunicefix mips
  echo " "
  case "$usrinc" in
  '') ;;
